src/Components/Home.js

```diff
@@ -0,0 +1,75 @@
+import { NavLink, Link } from "react-router-dom";
+import { allVideos } from "../Database";
+import { useVideo } from "../Contexts";
+export const Home = () => {
+  return (
+    <div className="home-wrapper">
+      <aside className="home-wrapper__aside">
+        <ul className="home-wrapper__aside__list">
+          <NavLink to="/" className="side-nav-link">
+            <li className="pointer">
+              <i className="fas fa-home pointer"></i>
+              <span className="list-item-title">Home</span>
+            </li>
+          </NavLink>
+
+          <NavLink
+            to="/watchHistory"
+            className="side-nav-link"
+            
+          >
+            <li className="pointer">
+              <i class="fas fa-history pointer"></i>
+              <span className="list-item-title">History</span>
+            </li>
+          </NavLink>
+
+          <NavLink to="/playlists" className="side-nav-link">
+            <li className="pointer">
+              <i class="fas fa-list pointer"></i>
+              <span className="list-item-title">Playlists</span>
+            </li>
+          </NavLink>
+
+          <NavLink to="/watchLaterVideos" className="side-nav-link">
+            <li className="pointer">
+              <i class="fas fa-clock pointer"></i>
+              <span className="list-item-title">Watch later</span>
+            </li>
+          </NavLink>
+
+          <NavLink to="/likedVideos" className="side-nav-link">
+            <li className="pointer">
+              <i class="fas fa-thumbs-up pointer"></i>
+              <span className="list-item-title">Liked videos</span>
+            </li>
+          </NavLink>
+        </ul>
+      </aside>
+      <main className="home-wrapper__main">
+        {allVideos.map((item) => {
+          return (
+            <Link to={`/video/${item.id}`} className="video-item-link pointer">
+              <div className="video-item" key={item.id}>
+                <img
+                  style={{ width: "100%", height: "50%" }}

```

Comment:
> Hey Danish!
> Video Library looks good.
> I've added comments to your code, hope they help!
> Overall, I would suggest better indentation of your code and also naming variables better for understanding.
> Eg. With "playlists" and "playlist", it's hard to understand what is stored in the variables while going through code.
> Otherwise LGTM, good job.

Thanks a lot Neha. Yes I will improve the naming for sure !

Reply:
> Width and height can be set as constant variables and set using className or you can move these to the css file.

Yes i will move them to an ext file